appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -1789,10 +1789,12 @@
           360))
 
 (define (sin-degrees degrees)
-  (sin (degrees->radians-internal degrees)))
+  (if (= mod 180) 0
+    (sin (degrees->radians-internal degrees)))
 
 (define (cos-degrees degrees)
-  (cos (degrees->radians-internal degrees)))
+  (if (or (= mod 90) 0
```

Comment:
This change won't work because it will result in `cos(0) = 0` whereas `cos(0) = 1` is the right answer. See my earlier comment for the correct test.